packages/lesswrong/server/reviewVoteUpdate.ts

```diff
@@ -304,3 +288,45 @@ export async function createVotingPostHtml () {
 //   .sort({reviewVoteScoreAllKarma:-1})
 
 // MAKE SURE TO UPDATE LIMIT OF QUERY IN UI
+
+
+// SELECT 
```

Comment:
cleanup

Reply:
I'm cleaned up some things around here but not 100% sure what you were referring to. The sql is left there deliberately (although I removed the old noSqlBooster code)